README.md

```diff
@@ -357,6 +357,32 @@ stack exec vaultenv -- --token SECRET --secrets-file foo.env /usr/bin/env
 It is possible to `stack build` with `--split-objs` to produce a smaller binary.
 To take full advantage of this, the Stackage snapshot has to be rebuilt.
 
+If you want a static binary, you can install [Nix](https://nixos.org/nix/) and
```

Comment:
I've emphasized this and mentioned `libc`

I've left mentions `libgmp` out because I would otherwise also have to note that we build with `integer-simple` which is not the point of this section of the docs